docs/blitz/design/actions/yang.rst

```diff
@@ -100,17 +100,20 @@ subscription, or, you may expect the test to fail (referred to as a negative tes
 .. code-block:: YAML
 
     format:
-      request_mode:    # [STREAM, ONCE, POLL] gNMI subscription mode
-      sub_mode:        # [ON_CHANGE, SAMPLE] gNMI subscription sub_mode
-      encoding:        # [JSON, JSON_IETF, PROTO, ASCII] gNMI val encoding
-      prefix:          # [true | false] gNMI message requires PATH prefix
-      origin:          # [openconfig | rfc7951 | module | <device defined> ] gNMI origin
-      base64:          # [true | false] gNMI set "val" requires Base64 encoding
-      sample_interval: # number of seconds between sampling
-      stream_max:      # seconds to stop stream (default: 0, no max)
-      auto-validate:   # [true | false] automatically validate config messages
-      negative-test:   # [true | false] expecting device to return an error
-      pause:           # pause N seconds between each test (default: 0, no pause)
+      request_mode:     # [STREAM, ONCE, POLL] gNMI subscription mode
+      sub_mode:         # [ON_CHANGE, SAMPLE] gNMI subscription sub_mode
+      encoding:         # [JSON, JSON_IETF, PROTO, ASCII] gNMI val encoding
+      prefix:           # [true | false] gNMI message requires PATH prefix
+      origin:           # [openconfig | rfc7951 | module | <device defined> ] gNMI origin
+      base64:           # [true | false] gNMI set "val" requires Base64 encoding
+      sample_interval:  # number of seconds between sampling
+      stream_max:       # seconds to stop stream (default: 120, no max)
+      auto-validate:    # [true | false] automatically validate config messages
+      negative-test:    # [true | false] expecting device to return an error
+      pause:            # pause N seconds between each test (default: 0, no pause)
+      transaction_time: # number of seconds that determines the maximum time that can pass between sending a request and receiving a response
+      polls_number:     # only for POLL subscribtions, determines how many polls should be sended (default: stream_max // sample_interval)
```

Comment:
"``sample_interval``" and "``polls_number``" are very similar.  The only difference is ``sample`` is a push from the device and ``poll`` is a pull from the device.  Rather than adding "``polls_number``" change "``sample_interval``" to "``sample_poll``".
```
sample_poll:       # number of seconds between SAMPLE sub_mode or POLL request_mode
```
The poll should be measured in seconds meaning how many seconds to wait before the next poll.  Same as sample.

Reply:
Changing sample_interval, will break many existing tests so I think we can stay with sample_interval and just use it in POLL like you said.

Reply:
You can easily make sample_interval backward compatible without breaking any tests.
```
sample_poll = request.get('sample_poll', {}).get('sample_interval')
```
Once we take "``sample_interval``" out of the documentation "``sample_poll``" will be the standard for new tests.